Project.toml

```diff
@@ -14,5 +14,5 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 [compat]
 FastClosures = "0.2.1, 0.3.0"
 ForwardDiff = "0.9.0, 0.10.0"
-LinearOperators = "0.5.0, 0.6.0"
+LinearOperators = "0.7.0, 2"
```

Comment:
The syntax is just "0.7.0" now.
"0.7.0, 2" means "0.7.x" and "2.x".

Reply:
Actually "0.7.0, 1.0" to support 1.0 as well.

Reply:
If I understand [the docs](https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1) correctly, "0.7.0, 2" means "from 0.7.0 up to, and not including, 2.0.0".

Reply:
I guess it should be "0.7.0, 1".